lib/omise/OmiseCharge.php

```diff
@@ -97,11 +97,16 @@ public function expire()
     /**
      * Captures a charge.
      *
+     * @param int|null $captureAmount  optional capture amount.
      * @return OmiseCharge
      */
-    public function capture()
+    public function capture(?int $captureAmount = null)
```

Comment:
I think type declaration for `int` do not support in PHP`< 7.0.0`.
And we trying to support from 5.6, so may be we remove type declaration here?
cc : @aashishgurung